WalletWasabi.Gui/ViewModels/WasabiDocumentTabViewModel.cs

```diff
@@ -71,13 +75,32 @@ public virtual void OnDeselected()
 			IsSelected = false;
 		}
 
-		public virtual void OnOpen()
+		void IDockableViewModel.OnOpen() // This interface member is called explicitly from Avalonia after the Tab was opened.
```

Comment:
```suggestion
		// This interface member is called explicitly from Avalonia after the Tab was opened.
		void IDockableViewModel.OnOpen()
```

I think it is better to `Place the comment on a separate line, not at the end of a line of code.`